### PR TITLE
[Buckinghamshire] Nicer not-our-road messages.

### PR DESCRIPTION
**Pull request by dracos**
Fixes https://github.com/mysociety/fixmystreet-freshdesk/issues/37
[skip changelog]

### CODE REVIEW

File: web/cobrands/buckinghamshire/assets.js [hunk 1]
```diff
@@ -242,9 +252,16 @@ fixmystreet.assets.add($.extend(true, {}, defaults, {
                 enable_report_form();
             } else {
                 // User has clicked a road that Bucks don't maintain.
+
+                var map = {
+                    "HE": '#js-not-council-road-he',
+                    "HWOA": '#js-not-council-road-other'
```
Comment (davea) [inline web/cobrands/buckinghamshire/assets.js]: Is this missing `P`/`HWSA`? Clicking at, e.g. `/report/new?longitude=-0.8269637409057568&latitude=51.819437574604656` doesn't result in the right message being shown:
image

File: templates/web/buckinghamshire/report/new/roads_message.html [hunk 2]
```diff
@@ -1,8 +1,22 @@
 <div id="js-roads-responsibility" class="box-warning hidden">
-    <strong>Invalid location</strong>
-    <div id="js-not-council-road" class="hidden js-responsibility-message">
-        <p>The selected road is not maintained by Buckinghamshire County Council.</p>
-        <p>Please select another road.</p>
+    <strong>Not maintained by Buckinghamshire County Council</strong>
+    <div id="js-not-council-road-private" class="hidden js-responsibility-message">
```
Comment (davea) [inline templates/web/buckinghamshire/report/new/roads_message.html]: Alternatively to the other comment, I guess this should be `id="js-not-council-road"`?

File: templates/web/buckinghamshire/report/new/roads_message.html [hunk 3]
```diff
@@ -1,8 +1,22 @@
...
+    <div id="js-not-council-road-other" class="hidden js-responsibility-message">
+        <p>The selected road is not maintained by Buckinghamshire County Council,
+        but by a neighbouring council. Please move your pin slightly, to the
+        other side of the border.
+        </p>
+    </div>
+    <div id="js-not-council-road-he" class="hidden js-responsibility-message">
+        <p>The selected road is not maintained by Buckinghamshire County Council,
+        but by Highways England. Please follow this link to
+        <a class="js-update-coordinates" href="https://www.fixmystreet.com/report/new?latitude=[% latitude |html %]&amp;longitude=[% longitude | html %]">FixMyStreet</a> to continue reporting your issue here.
```
Comment (davea) [inline templates/web/buckinghamshire/report/new/roads_message.html]: This is neat 💯

File: templates/web/buckinghamshire/report/new/roads_message.html (hunk 2 above)
Comment (dracos) [reply templates/web/buckinghamshire/report/new/roads_message.html]: Aha, yes, whoops, ta.

**Pull request merged**